modules/gdscript/gdscript.cpp

```diff
@@ -2213,13 +2212,13 @@ GDScriptLanguage::GDScriptLanguage() {
 	}
 
 #ifdef DEBUG_ENABLED
-	GLOBAL_DEF("debug/gdscript/warnings/enable", true);
-	GLOBAL_DEF("debug/gdscript/warnings/treat_warnings_as_errors", false);
-	GLOBAL_DEF("debug/gdscript/warnings/exclude_addons", true);
+	GLOBAL_DEF(PropertyInfo(Variant::BOOL, "debug/gdscript/warnings/enable"), true);
+	GLOBAL_DEF(PropertyInfo(Variant::BOOL, "debug/gdscript/warnings/treat_warnings_as_errors"), false);
+	GLOBAL_DEF(PropertyInfo(Variant::BOOL, "debug/gdscript/warnings/exclude_addons"), true);
```

Comment:
Some of these defs got longer, what's the justification for this? Consistency? Readability?

Reply:
I had to do that in order to use PropertyInfo for every usage of `GLOBAL_DEF`. The other option was to have two macros for each variation of `GLOBAL_DEF`, one that accepts a PropertyInfo and one that accepts a string like it did before this PR. There are 8 variations of `GLOBAL_DEF` though, which is why I opted to make all of them use PropertyInfo instead.